docs/source/release_notes.rst

```diff
@@ -33,6 +34,7 @@ Release Notes
 
     **Breaking Changes**
         * Updated ``start_iteration_callback`` to accept a pipeline instance instead of a pipeline class and no longer accept pipeline parameters as a parameter :pr:`2290`
+        * Moved ``default_parameters`` to ``ComponentGraph`` from ``PipelineBase``. A pipeline's ``default_parameters`` is now accessible via ``pipeline.component_graph.default_parameters`` :pr:`2307`
```

Comment:
Yup, you're right! 

Filed https://github.com/alteryx/evalml/issues/2331, gonna work on it in a separate branch c:
(I believe its because we relied on the input component graph aka `component_graph` to determine if linear but also wanted to store `_component_graph` aka the underlying graph structure.)